backend/tests/test_integration_org_unit.py

```diff
@@ -2832,3 +2832,68 @@ def test_tree(self):
         for path, expected in util.get_fixture('test_trees.json').items():
             with self.subTest(path):
                 self.assertRequestResponse(path, expected)
+
+    def _create_conf_data(self):
+        import sqlite3
+        import mora.settings as settings
+        import os
+        os.remove(settings.USER_SETTINGS_DB_FILE)
+
+        defaults = {'show_roles': 'True',
+                    'show_user_key': 'False',
+                    'show_location': 'True'}
+
+        with sqlite3.connect(settings.USER_SETTINGS_DB_FILE,
+                             check_same_thread=False) as conn:
+            cur = conn.cursor()
+
+            create_query = ('CREATE TABLE orgunit_settings ' +
+                            '(id INTEGER PRIMARY KEY AUTOINCREMENT, ' +
+                            'object UUID, setting varchar(255) NOT NULL, '
+                            'value varchar(255) NOT NULL);')
+            cur.execute(create_query)
+
+            query = ('INSERT INTO orgunit_settings (object, setting, value) ' +
+                     'VALUES (Null, ?, ?);')
+            for setting, value in defaults.items():
+                cur.execute(query, (setting, value))
+        return True
+
+    def test_global_user_settings_read(self):
+        self._create_conf_data()
+        url = '/service/o/get_configuration'
+
+        user_settings = self.assertRequest(url)
+        print(user_settings)
+        self.assertTrue('show_location' in user_settings)
+        self.assertTrue('show_user_key' in user_settings)
+        self.assertTrue('show_roles' in user_settings)
+        self.assertTrue(user_settings['show_location'] == 'True')
+
+    def test_global_user_settings_write(self):
+        postgres_url = self._create_conf_data()
```

Comment:
Get rid of this variable assignment as it's de-facto unused, given that you're injecting it into something that contains no formatting directives.